velox/exec/TreeOfLosers.h

```diff
@@ -230,7 +231,7 @@ class TreeOfLosers {
         return left;
       } else {
         values_[node] = left.first;
-        equals_[node] = right.second;
+        equals_[node] = left.second;
```

Comment:
Nice catch

Reply:
Would it be possible to add unit test for this fix?

Reply:
Yeah, per our offline discussion, I plan to add test to cover this in followup PR with a dedicated test case. And will verify if it triggers this or not.